common/lib/xmodule/xmodule/library_sourced_block.py

```diff
@@ -41,38 +45,54 @@ class LibrarySourcedBlock(StudioEditableXBlockMixin, EditableChildrenMixin, XBlo
         display_name=_("Display Name"),
         scope=Scope.content,
     )
-    source_block_id = String(
-        display_name=_("Library Block"),
+    source_block_ids = List(
```

Comment:
Note that  this makes the block incompatible with the existing implementation (to keep the code neater), as I'd assumed that this isn't being used in any production environment. 
I can rename this back to `source_block_id` and wrap str in a list if compatibility needs to be preserved.

Reply:
AFAIK It's not being used in prod yet, breaking changes are fine.